src/space-picker/space-picker.js

```diff
@@ -33,7 +33,7 @@ const Self = class SpacePicker extends ColorElement {
 		if (name === "spaces") {
 			if (!this.groups) {
 				this._el.picker.innerHTML = Object.entries(this.spaces)
-					.map(([id, space]) => `<option value="${id}">${space.name}</option>`)
+					.map(([id, space]) => `<option label="${ this.getSpaceLabel(space) }" value="${ id }">${ space.name }</option>`)
```

Comment:
Why are we using all three here? Just put the call to `this.getSpaceLabel()` in the content.

Reply:
That's an excellent point! It's such a blessing to have more than one pair of eyes. Thank you! I'll fix it.